pulpcore/app/tasks/telemetry.py

```diff
@@ -0,0 +1,45 @@
+import os
+from django.db.models import Sum
+
+from pulpcore.app.models import Artifact
+
+from opentelemetry.sdk.metrics import MeterProvider
+
+from opentelemetry.exporter.otlp.proto.http.metric_exporter import OTLPMetricExporter
+from opentelemetry.sdk.metrics.export import PeriodicExportingMetricReader
+from opentelemetry.sdk.resources import Resource
+
+# This configuration is needed since the worker thread is not using the opentelemetry
+# instrumentation agent to run the task code.
+
+OTLP_EXPORTER_ENDPOINT = os.getenv("OTEL_EXPORTER_OTLP_ENDPOINT", "http://localhost:4318/")
+
+exporter = OTLPMetricExporter()
+
+resource = Resource(attributes={"service.name": "pulp-worker"})
+
+metric_reader = PeriodicExportingMetricReader(
+    exporter, export_interval_millis=3000, export_timeout_millis=3000
+)
+provider = MeterProvider(metric_readers=[metric_reader], resource=resource)
+
+
+def emit_domain_space_usage_metric():
```

Comment:
The only need for more than one such task would be the typical "Hourly, Daily, Weekly" schedules.
```suggestion
def otel_metrics():
```

Reply:
I don't get it. The name make it clear what the function does. The `telemetry` module could even receive the code of the task metrics we have in `worker.py`.
I see no reason to change the name here.